### PR TITLE
Update Set-SPOUser.md

### PR DESCRIPTION
**Pull request by PowershellScripts**
suggestion

### CODE REVIEW

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOUser.md [hunk 1]
```diff
@@ -92,7 +92,7 @@ Accept wildcard characters: False
 ```
 
 ### -UpdateUserTypeFromAzureAD
-If the UserType property of an User is changed in AzureAD from Member to Guest and vice-versa, this parameter can be used to update it in SharePoint Online.
+If the UserType property of an existing user is changed in AzureAD from Member to Guest and vice-versa, this parameter can be used to update it in SharePoint Online.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOUser.md:95]:
```suggestion
If the UserType property of an existing user is changed in Azure AD from Member to Guest and vice-versa, this parameter can be used to update it in SharePoint Online.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @PowershellScripts, kindly commit the minor additional suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOUser.md [hunk 2]
```diff
@@ -92,7 +92,7 @@ Accept wildcard characters: False
 ```
 
 ### -UpdateUserTypeFromAzureAD
-If the UserType property of an User is changed in AzureAD from Member to Guest and vice-versa, this parameter can be used to update it in SharePoint Online.
+If the UserType property of an existing user is changed in Azure AD from Member to Guest and vice-versa, this parameter can be used to update it in SharePoint Online.
```
Comment (tseward) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOUser.md:95]: I would change this to "Guest or vice-versa,".

**Review by tseward — CHANGES_REQUESTED** [review]

**Review by dariomws — APPROVED** [review]

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOUser.md [hunk 3]
```diff
@@ -92,7 +92,7 @@ Accept wildcard characters: False
 ```
 
 ### -UpdateUserTypeFromAzureAD
-If the UserType property of an existing user is changed in AzureAD from Member to Guest and vice-versa, this parameter can be used to update it in SharePoint Online.
+If the UserType property of an existing user is changed in Azure AD from Member to Guest and vice-versa, this parameter can be used to update it in SharePoint Online.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOUser.md:95]:
```suggestion
If the UserType property of an existing user is changed in Azure AD from Member to Guest or vice-versa, this parameter can be used to update it in SharePoint Online.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@PowershellScripts, kindly commit the suggested change.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tseward

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**